composer.json

```diff
@@ -32,8 +32,9 @@
         "symfony-scripts": [
             "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
+            "@php bin/console cache:clear --no-warmup --ansi",
+            "@php bin/console cache:warmup --ansi",
+            "@php bin/console assets:install web --symlink --relative --ansi",
```

Comment:
I'm 👎 on this change for now as it looses the ability to support extra config from the `composer.json`.